src/registry.rs

```diff
@@ -63,13 +65,52 @@ impl RegistryCore {
         // throughout the lifetime of a program.
         Ok(())
     }
+
+    #[allow(unknown_lints)]
```

Comment:
I think it wants you to use `mf_by_name.entry` instead of `mf_by_name.contains`.